include/internal/bio.h

```diff
@@ -24,3 +26,8 @@ struct bio_method_st {
 
 void bio_free_ex_data(BIO *bio);
 void bio_cleanup(void);
+
+
+/* Old style to new style BIO_METHOD conversion functions */
+int bwrite_conv(BIO *bio, const char *in, size_t inl, size_t *written);
+int bread_conv(BIO *bio, char *out, size_t outl, size_t *read);
```

Comment:
Similar to the public API changes, I would like to see `in` and `out` changed to `data`, and `inl` and `outl` to `datal`.